Document-Processing/Excel/Spreadsheet/Javascript-ES6/comment.md

```diff
@@ -0,0 +1,191 @@
+
+### Resolve a comment
+* In the comment container, click the **⋯ (More thread actions)** menu in the container header and select **Resolve Thread**. 
+
+### Reopen a comment
+* In the comment container, click the **Reopen** button in the container header to make the thread active again.
+
+![Resolve and reopen in Spreadsheet](./images/spreadsheet_resolve_reopen.gif)
+
+You can also use the `isResolved` property in the comment model when initializing or updating comments programmatically.
+
+## Deleting a comment or reply
+You can delete either a specific reply or an entire comment thread (including all replies) using the following options:
+
+### Deleting a comment thread
+* **Context menu**: Right-click the cell that contains the comment and select **Comment > Delete Comment**.
+* **Ribbon**: Use **Review > Comment > Delete Comment** on a cell that contains the comment.
+* **Comment container**: In the comment container, click the **⋯ (More thread actions)** menu in the header and select **Delete Thread** for an active comment or use the **Delete Thread** button in container header for a resolved comment.
+
+![Deleting a comment thread in Spreadsheet](./images/spreadsheet_delete_comment.png)
+
+Deleting a thread removes the comment and all its replies from the cell.
+
+### Delete a reply
+In the comment container, hover over the reply and click the **⋯ (More actions)** menu then select **Delete Comment**.
+
+![Deleting a comment reply in Spreadsheet](./images/spreadsheet_delete-reply_comment.png)
+
+## Next and Previous Comment
+
+The **Review > Comment > Next Comment and Previous Comment** options in the ribbon allows you to quickly navigate between cells that contain comments:
+
+* **Next Comment**: Moves to the next cell with a comment.
+* **Previous Comment**: Moves to the previous cell with a comment.
+
+Navigation starts within the active sheet. When all comments in the active sheet have been visited (end or start reached), the navigation automatically continues to the next or previous sheet that contains comments. This ensures you can review all comments across the workbook without manually switching sheets.
+
+## Show Comments and Comments Review Pane
+
+The **Comments Review Pane** provides a centralized view of all comments in the active sheet and offers filtering, inline actions, and navigation. This pane helps you to manage comments efficiently without switching between individual cells.
+
+You can show or hide the Comments Review Pane using:
+
+* **Ribbon**: Use **Review > Comment > Show Comments**.
+* **Property**: Set the `showCommentsPane` property when initializing the Spreadsheet. The default value for the `showCommentsPane` property is `false`.
+
+![Show comments in Spreadsheet](./images/spreadsheet_show_comments.gif)
+
+### Features of the Comments Review Pane
+
+The Comments Review Pane allows you to:
```

Comment:
The "Comments" review pane allows you to:

Make it more elaborative in introduction.